1.1.0 Release (#32)
unsplash/datasets: DOCS.md

| photo_image_url                | URL of the image file. Note: this is a [dynamic URL](https://unsplash.com/documentation#dynamically-resizable-images), so you can apply [resizing and customization operations directly on the image](https://unsplash.com/documentation#supported-parameters) |
| photo_submitted_at             | Timestamp of when the photo was submitted to Unsplash |
| photo_featured                 | Whether the photo was promoted to the [Editorial feed](https://unsplash.com/) or not |

> | photo_width                    | Width of the photo |
> | photo_height                   | Height of the photo |

Reviewer: ```suggestion
| photo_width                    | Width of the photo in pixels |
| photo_height                   | Height of the photo in pixels |
```

| anonymous_user_id             | Anonymous user ID |
| conversion_country            | Country code of the device geolocation |

## 5 - colors.tsv

*Note: The coverage and score data comes from a 3rd party AI*

> The `colors.tsv` dataset has one row per major color present in the photo. The dataset tells you which photo contains which colors and by how much its covering it as well as how much focus the color has (score).

Reviewer: Maybe:

```suggestion
The `colors.tsv` dataset has one row per major color present in the photo. The dataset tells which colors are contained within a photo, their coverage as a percentage, and a score for how in focus the color is.
```
